### ISSUE TITLE
Captcha on New Thread

### ISSUE THREAD

**Issue by MagicalCat1172** [kyb3r/modmail#2846]
I am always tired of selfbots that likes to spam the modmails. If there is a way to make the users do a captcha before creating a thread that will be great. 
The current alternative is just to set the guild join higher and higher. But it has some bad side effects like real people who actually have question, not being able to request for support.
I am pretty sure this will benefit a lot of people.
I already seen other modmail bots have this feature.

**Comment by StephenDaDev** [comment]
I do agree that self bots spamming modmails, is quite annoying. I also think you spamming modmails is quite annoying.

**Comment by DAzVise** [comment]
Yeah setting guild_age might have some side effects, but you could set the account_age instead? There are very low chances someone creates an account and joins your server and needs to DM Modmail immediately

**Comment by MagicalCat1172** [comment]
> I do agree that self bots spamming modmails, is quite annoying. I also think you spamming modmails is quite annoying. _cough the Modmail Official one_

??? I only sent that several messages asking for some help, before you decided to ban me.

**Comment by Shashank3736** [comment]
> Yeah setting guild_age might have some side effects, but you could set the account_age instead? There are very low chances someone creates an account and joins your server and needs to DM Modmail immediately

Actually setting account_age is not 100% secure because self bots now days have over 90 days account age. Captcha is the best solution for this.

**Comment by sumit309** [comment]
Hey
